runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -164,6 +183,11 @@ void update(boolean updated, XpSnapshotSingle xpSnapshotSingle)
 
 	private void rebuildAsync(boolean updated, XpSnapshotSingle xpSnapshotSingle)
 	{
+		if (pause.getText().equals("Resume"))
```

Comment:
I don't like this text equality as state

Reply:
I agree it's not my favorite thing to do. It seems robust however and considering the current code is structured in a somewhat different way I didn't want to alter that too much. Considering this works, is it really an issue?

Reply:
If you manage to land this, I will change as soon as I have energy to.

Reply:
I'm down with restructuring the plugin code to make it a little better for pausing purposes if you feel that is necessary. Just initially thought the effort required to do that wouldn't really be worth it considering this solution is fine just a little ugly.